docs/index.md

```diff
@@ -2,11 +2,13 @@
 
 --8<-- "docs/.partials/index-header.html"
 
-When I first found FastAPI, I got it immediately. I was excited to find something so innovative and ergonomic built on Pydantic.
+**PydanticAI is a Python Agent Framework designed to make it less painful to build production grade applications with Generative AI.**
```

Comment:
```suggestion
**PydanticAI is a Python Agent Framework designed to make it less painful to build production-ready applications with Generative AI.**
```

Reply:
changed to `agent framework`, kept `production grade`.